### DISCUSSION TITLE
On which repository assignment should be pull requested?

### DISCUSSION THREAD

**Discussion by VaishnavGhenge** [open-sauced-craftwork/browser-extension#43] [Q&A] [answered]
After opting for open sauced browser extension assignment I have given a private repository so I cannot pull request to it as I can't fork new branch out of it to my GitHub account.

So, I forked new repository out of main branch `browser-extension` repo to my GitHub account and cloned it to my local machine. Now my forked repo has my final assignment. What should I do next? or please correct if I am wrong in understanding steps.

**Comment by anush008** [comment, marked as answer]
You can follow the steps mentioned here #34.
